### DISCUSSION TITLE
Set proxy env variables for package runtime containers via deploymentRuntimeConfigs

### DISCUSSION THREAD

**Discussion by exolicious** [crossplane/crossplane#5989] [Q&A]
Hi guys, i want to do what the title suggests.
I have a problem when installing functions in an airgapped environment that runs behind a proxy and tries to fetch that functions image, if i dont set the environment variable.
Currently I have a solution which requires me to set the "extraEnvVarsCrossplane" value in the helm chart values file.
However, that also applies the environment variables to the runtime image of the crossplane provider pod resulting in lookup timeouts when trying to access the kubeapi / grpc services of the function pods.
The solution to this is setting theno_proxy env variable to list the **IPs** of the kubeapi and the function pods, which is a terrible workaround.

What I want to do:
Create a new DeploymentRuntimeConfig with the proxy variables and reference that through the spec.runtimeConfigRef of the Function i want to install, but that does not seem to work?
Is the crossplane pod the one fetching the function packages?

Any ideas, as to how to get this setup running?

**Comment by gregoryvortman** [comment]
I just added proxy variable on top of helm install command with following option: 
--set extraEnvVarsCrossplane.HTTP_PROXY=http://<proxy_host>:<proxy_port>
works fine for me
